### DISCUSSION TITLE
My contributions

### DISCUSSION THREAD

**Discussion by wytten** [community/community#114237] [Other Feature Feedback, Questions, & Ideas]
I'd like to see my contributions so that I can gaze on my recent history in sort of a sparkline format (how much did I work this weekend, really?)

**Comment by jfullstackdev** [comment]
it's already shown

![image](https://github.com/community/community/assets/47092464/3acd9409-ce59-49e5-8376-8390b5b51448)

you may want to click the Setting button to `include private contributions` too

**Comment by arcticsoftxyz** [comment]
Go to your GitHub profile. You'll see a heatmap-style calendar showing your contributions over the past year. 
Each square represents a day, and the color intensity indicates the number of contributions.

![Screenshot 2025-01-30 004552](https://github.com/user-attachments/assets/979fba52-2b66-43d7-a940-f4e299634549)


Also, you need to explicitly **enable private contributions** in your GitHub settings.

1. Go to your GitHub profile.
2. Click on "Settings".
3. In the left sidebar, click on "Contributions".
4. Make sure the option "Include private contributions on my profile" is enabled.

![Screenshot 2025-01-30 004655](https://github.com/user-attachments/assets/d8ef9133-326c-482d-bd1c-4b341db7dc4f)
